other/js/auth_plugin_examples.js

```diff
@@ -47,8 +47,8 @@ function urlTokenMatch(url, token, verbose=false) {
 exports.tokenAuth = function tokenAuth(source) {
     /**
      * Authorisation plugin which validates the token query parameter against
-     * a token provided as the argument to the --auth-source command line
-     * argument
+     * a token provided as the argument to the `--auth-source` command line
+     * argument.
```

Comment:
Could you squash the changes from this commit (22ffd2c) into the commit that originally introduced these comments please